### PR TITLE
Merge develop into master 

### PR DESCRIPTION
**Pull request by clausmullie**
Would be good to have the evaluation criteria on there to discuss with members

### CODE REVIEW

**Comment by clausmullie** [comment]
- [ ] writing-issues.md doesn't seem to be linked from anywhere

**Comment by bvhme** [comment]
FYI, this creates a regression on #312 with the missing phone number

**Comment by clausmullie** [comment]
@bvhme I don't quite understand your comment - #312 states nothing about phone numbers, and these changes do not affect the contact page which has the number for ambi. But I think you mean the number on should remain on the hiring pages, so we've readded this #321.

**Comment by bvhme** [comment]
Sorry for the confusion, I must not have found the correct issue.

What I meant to say is that my number for now should remain on at least the
contact page for the ANBI requirements.

**Comment by clausmullie** [comment]
@bvhme - requires you to update branch

File: activities/codebase-stewardship/criteria-for-codebase-stewardship.md [hunk 1]
```diff
@@ -14,3 +16,63 @@ Here are the criteria the staff of the Foundation for Public Code will use to he
...
+1. How long would it take a team of 2 devs to extract to make the code configurable such that it could be deployed for a different org?
+2. What languages are used in your project?
+
+### Reuse potential
+
+1. What portion of the codebase is only understood by 1 or 2 people?
+2. Is the relevant policy bundled with the code?
+3. Can the relevant policy be bundled with the code in English?
+
+### Societal impact
```
Comment (bvhme) [inline activities/codebase-stewardship/criteria-for-codebase-stewardship.md]: Sorry for the oversight, but here are headings without content, which should be removed.

File: activities/member-relations/index.md [hunk 2]
```diff
@@ -8,12 +8,13 @@ This outlines the stages by which a public organization becomes a founding membe
...
 
-At this stage, the public organization confirms their interest in becoming a member by signing the [expression of interest](expression-of-interest.md).
+At this stage, the public organization confirms their interest in becoming a member by signing the [expression of interest](expression-of-interest.md). This grants both parties the right to discuss this collaboration publicly. It's signed by a policy lead and/or technical development lead in the public organization.
 
-This grants both parties the right to discuss this collaboration publicly. It's signed by a policy lead and/or technical development lead in the public organization.
+This stage may include the '[Information about Founding Membership](information-about-founding-membership.md)'.
 
 This stage is over when the expression of interest has been signed.
 
+
```
Comment (bvhme) [inline activities/member-relations/index.md]: There is an extra whitespace here.

File: activities/member-relations/index.md [hunk 3]
```diff
@@ -8,12 +8,13 @@ This outlines the stages by which a public organization becomes a founding membe
 
 ## 1. Expression of interest
 
-At this stage, the public organization confirms their interest in becoming a member by signing the [expression of interest](expression-of-interest.md).
+At this stage, the public organization confirms their interest in becoming a member by signing the [expression of interest](expression-of-interest.md). This grants both parties the right to discuss this collaboration publicly. It's signed by a policy lead and/or technical development lead in the public organization.
 
-This grants both parties the right to discuss this collaboration publicly. It's signed by a policy lead and/or technical development lead in the public organization.
+This stage may include the '[Information about Founding Membership](information-about-founding-membership.md)'.
```
Comment (bvhme) [inline activities/member-relations/index.md]: This is fine. But as for good form this link should not be in quotes and can be more descriptive.

File: activities/member-relations/information-about-founding-membership.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+type: Resource
+---
+
+# Information about Founding Membership template
+
+## Resource
+
+You can access the [template here](https://docs.google.com/document/d/165RXZmeU6klU_QmCR06ACvY8SAtN27de1nilqOmi8Nw/edit?usp=sharing).
```
Comment (bvhme) [inline activities/member-relations/information-about-founding-membership.md]: This is fine, but for good form (and printing this with relevant styling) you should make the link a description of what it is, e.g.

```suggestion
The template is in [the Google Doc "Template_Information about Founding Membership"](https://docs.google.com/document/d/165RXZmeU6klU_QmCR06ACvY8SAtN27de1nilqOmi8Nw/edit?usp=sharing).
```

File: activities/member-relations/information-about-founding-membership.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+* This lightweight document that can be shared as minutes after an introductory meeting with interested public organizations
+* This resource is a link to a Google doc for now, but might change format
+* Any proposed changes to the content of this template must first be proposed on, and merged with, [the `master` branch of About](https://github.com/publiccodenet/about/tree/master) before being implemented here
+
+## Use and further development
+
+- [ ] Replace the “[highlighted text]” with customized text
+- [ ] Share with potential members
+- [ ] Revise template with feedback once shared with 2-3 potential members
```
Comment (bvhme) [inline activities/member-relations/information-about-founding-membership.md]: This should be tracked elsewhere and mixing usage and development instructions adds a lot of extra strain on both users and developers.

**Comment by clausmullie** [comment]
Thanks for review @bvhme, fixes done with #329

File: activities/member-relations/information-about-founding-membership.md (hunk 4 above)
Comment (clausmullie) [reply activities/member-relations/information-about-founding-membership.md]: see #329

File: activities/member-relations/index.md (hunk 3 above)
Comment (clausmullie) [reply activities/member-relations/index.md]: see #329

File: activities/member-relations/index.md (hunk 2 above)
Comment (clausmullie) [reply activities/member-relations/index.md]: see #329

File: activities/codebase-stewardship/criteria-for-codebase-stewardship.md (hunk 1 above)
Comment (clausmullie) [reply activities/codebase-stewardship/criteria-for-codebase-stewardship.md]: see #329

**Pull request merged**